docs/development/ide-setup/intellij-setup.md

```diff
@@ -1,23 +1,27 @@
 ## *IntelliJ* Setup
 
-### Plugins:
-  - *Google Java Format*
-    - *settings > Other settings > google-java-format*
-    - Check 'enable'
+Setting up *IntelliJ* is pretty simple.  There are 4 plugins to install with the associated settings, and some settings to download and import.  
+
+### Plugins: 
+  1. *Google Java Format*
+        1. *settings > Other settings > google-java-format*
+        2.  Check 'enable'
       ![Screenshot from 2019-08-08 17-35-52
       ](https://user-images.githubusercontent.com/12397753/62746114-07cc2b80-ba03-11e9-9ac0-0b1e6e1e8788.png)
-  - *checkstyle-IDEA* [plugin](https://github.com/jshiell/checkstyle-idea)
-    - after install finish configuration in: **Other Settings > Checkstyle**
-    - select checkstyle file:
-  - *Save Actions*
-    - configure in settings to add 'final' to local variables and class variables.
-  - *Lombok*
+  2. *checkstyle-IDEA* [plugin](https://github.com/jshiell/checkstyle-idea)
```

Comment:
Nit, most markdown you can use `1` for each item in the numbered list and the numbering will be done automatically. This lets you insert items in the middle of the list without having to renumber the whole thing.

Reply:
I saw it here and one of your other comments, what is nit?

Reply:
"nit" is short for "nitpick". You will probably see that and 'side-comment' from reviewers. 'side-comment' means discussion about something that already existed and is not for the current PR.

Nits can be frustrating if a reivew is all nitpicks and is not looking at the larger picture. A comment prefaced as a nit is an acknowledgement that it's a pretty low-level type of thing being commented on. So while ideally we would not need to nitpick, it is code, and the details do add up and can be very important.